etc/login.defs

```diff
@@ -367,6 +368,19 @@ CHFN_RESTRICT		rwh
 #BCRYPT_MIN_ROUNDS 13
 #BCRYPT_MAX_ROUNDS 13
 
+#
+# Only works if ENCRYPT_METHOD is set to YESCRYPT.
+#
+# Define the YESCRYPT cost factor.
+# With a higher cost factor, it is more difficult to brute-force the password.
+# However, more CPU time and more memory will be needed to authenticate users
+# if this value is increased.
+#
+# If not specified, a cost factor of 8 will be used.
+# The value must be within the 1-11 range.
+#
+#YESCRYPT_COST_FACTOR 8
```

Comment:
This still says 8 in two places.

Reply:
I can't believe I forgot I defined this cost factor somewhere else >.<
Thank you.